### PR TITLE
Upgrade c2cgeoportal

### PR DESCRIPTION
**Pull request by sbrunner**
Commandes à effectuer pour migrer la base de donnée:

SQL:
```
UPDATE geov3.alembic_version SET version_num = '54645a535ad6';
CREATE SCHEMA geov3_static;
GRANT ALL ON SCHEMA geov3_static TO "www-data";
CREATE TABLE IF NOT EXISTS geov3_static.shorturl (
id serial PRIMARY KEY,
url character varying(1000),
ref character varying(20) NOT NULL UNIQUE,
creator_email character varying(200),
creation timestamp without time zone,
last_hit timestamp without time zone,
nb_hits integer
);
ALTER TABLE geov3_static.shorturl OWNER TO "www-data";
```

Bash:
```
.build/venv/bin/alembic -c alembic.ini upgrade head
.build/venv/bin/alembic -c alembic_static.ini upgrade head
```

### CODE REVIEW

**Comment by sbrunner** [comment]
fix #337

File: CONST_Makefile [hunk 1]
```diff
@@ -45,17 +43,17 @@ endif
...
-ifeq ($(PRINT3), TRUE)
+ifeq ($(PRINT_VERSION), 3)
 DEFAULT_BUILD_RULES += print
 endif
 BUILD_RULES ?= $(PRE_RULES) $(filter-out $(DISABLE_BUILD_RULES),$(DEFAULT_BUILD_RULES)) $(POST_RULES)
 
 # Requirements
 
-EGGS_DEPENDENTIES += .build/venv.timestamp setup.py CONST_versions.txt CONST_requirements.txt
+EGGS_DEPENDENTIES += .build/venv.timestamp-noclean setup.py CONST_versions.txt CONST_requirements.txt
```
Comment (pgiraud) [inline CONST_Makefile]: DEPENDENTIES -> DEPENDENCIES
Should be changed in c2cgeoportal probably.

**Comment by jaykayone** [comment]
can this be merged?

**Comment by sbrunner** [comment]
For me yes, I'm waiting for a review ...

**Comment by elemoine** [comment]
> For me yes, I'm waiting for a review ...

Have you seen @pgiraud's comment?

**Comment by sbrunner** [comment]
Yes, it's a c2cgeoportal issue that already corrected (thanks @pgiraud) https://github.com/camptocamp/c2cgeoportal/pull/1297, and will be corrected in the project with the next update ...

**Comment by elemoine** [comment]
+1 for merging this.

**Pull request merged**